corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ontonotes/CreateTrainDevTestSplit.java

```diff
@@ -0,0 +1,145 @@
+/**
+ * This software is released under the University of Illinois/Research and Academic Use License. See
+ * the LICENSE file in the root folder for details. Copyright (c) 2016
+ *
+ * Developed by: The Cognitive Computation Group University of Illinois at Urbana-Champaign
+ * http://cogcomp.cs.illinois.edu/
+ */
+package edu.illinois.cs.cogcomp.nlp.corpusreaders.ontonotes;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+
+import com.google.common.io.Files;
+
+import edu.illinois.cs.cogcomp.core.utilities.configuration.ResourceManager;
+import edu.illinois.cs.cogcomp.nlp.corpusreaders.corpusutils.CorpusSplitConfigurator;
+
+/**
+ * In an attempt to build a generic method to split data into test train and dev, this 
+ * class will traverse a directory distributing what data it finds into a test, train and
+ * dev set sequentially. The resulting data will be split across a test train and dev directory
+ * within an output directory provided on the command line. The split configuration is determined
+ * by the {@link edu.illinois.cs.cogcomp.nlp.corpusreaders.corpusutils.CorpusSplitConfigurator}.
```

Comment:
I could just as easily throw this away. I just needed a reasonable split for my own testing. I did look at the above referenced class, I don't recall why I didn't use it, but there was a reason.